core/templates/dev/head/domain/exploration/EditableExplorationBackendApiService.js

```diff
@@ -58,7 +55,7 @@ oppia.factory('EditableExplorationBackendApiService', [
       var editableExplorationDataUrl = (
         UrlInterpolationService.interpolateUrl(
         EDITABLE_EXPLORATION_DATA_URL_TEMPLATE, {
-          exploration_id: explorationId
+          exploration_id: String(explorationId)
```

Comment:
Currently it's only necessary due to tests in HistoryServiceSpec.js. In line 177, there is the section 
``` beforeEach(function() {
      mockExplorationData = {
        explorationId: 0
      };```
I can change this to a string and all tests pass. For the version however I must use String to convert from version Number(an int) to a string. Am doing that now.